schema/crdb/37.0.0/up02.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS live_repair_notification (
+    time TIMESTAMPTZ NOT NULL,
```

Comment:
It this timestamp the timestamp when the record was last updated?

Reply:
Yes